Examples/Sigmoid/SigmoidTest.py

```diff
@@ -1,150 +1,65 @@
-        model once adding terms starts to show diminishing returns. a good
-        default is 3 -- large datasets could benefit from higher values
-
-        'draws' is the total number of draws from the posterior for each tested
-        model
-
-        'draws' is the total number of draws from the posterior for each tested
-        
-         'threshav' is a threshold for proposing terms for elimination based on
-         their mean values (larger thresholds lead to more elimination)
-        
-         'threshstda' is a threshold standard deviation -- expressed as a fraction 
-         relative to the mean -- that pairs with 'threshav'.
-         terms with coefficients that are lower than 'threshav' and higher than
-         'threshstda' will be proposed for elimination (elimination will happen or not 
-         based on relative BIC values)
-        
-         'threshstdb' is a threshold standard deviation that is independent of the
-         mean value of the coefficient -- all with a standard deviation (fraction 
-         relative to mean) exceeding
-         this value will be proposed for elimination
-         
-        'gimmie' is a boolean causing the routine to return the most complex
-        model tried instead of the model with the optimum bic
-
-        'aic' is a boolean specifying the use of the aikaike information
-        criterion 
-    """
-# Running emulator routine and visualization, fit model to data
-betas, mtx, evs = model.fit(inputs, data)
-"""
-    inputs: 
-        'inputs' - normalzied inputs
-        
-        'data' - results
-    
-    outputs:
-         'betas' are a draw from the posterior distribution of coefficients: matrix, with
-         rows corresponding to draws and columns corresponding to terms in the GP
-        
-         'mtx' is the basis function interaction matrix from the
-         best model: matrix, with rows corresponding to terms in the GP (and thus to the 
-         columns of 'betas' and columns corresponding to inputs. a given entry in the 
-         matrix gives the order of the basis function appearing in a given term in the GP.
-         all basis functions indicated on a given row are multiplied together.
-         a zero indicates no basis function from a given input is present in a given term
-        
-         'ev' is a vector of BIC values from all of the models
-         evaluated
-"""
-
-meen, bounds, rmse = model.coverage3(inputs, data, draws, 1)
-"""
-    inputs:
-        'inputs' - normalized test inputs
-        'data' - test set results
-        'draws' - number of beta models used from model.fit()
-        plotting binary - 1 = show plot, 0 = do not show plot
-        
-    outputs:
-        'meen' - indexed predicted value
-        'bounds' - 95% confidence interval for each point
-        'rmse' - root mean square error
-"""
+from src.FoKL import FoKLRoutines # from FoKL import FoKLRoutines
```

Comment:
I don't believe from src.FoKL is necessary here FoKL package imports all the modules, should just be able to do 

```
from FoKL import FoKLRoutines
```

Reply:
double check this before committing

Reply:
correct -- my mistake leaving that in. fixed